vdr/didx509/resolver_test.go

```diff
@@ -227,6 +229,48 @@ func TestManager_Resolve_OtherName(t *testing.T) {
 		assert.ErrorIs(t, err, ErrNoMatchingHeaderCredentials)
 		metadata.JwtProtectedHeaders[X509CertThumbprintS256Header] = sha256Sum(signingCert.Raw)
 	})
+	t.Run("invalid signature of root certificate", func(t *testing.T) {
+		t.Skip("Can't test this right now, enable after https://github.com/nuts-foundation/nuts-node/issues/3587 has been fixed")
```

Comment:
this PR is claiming to close referenced issue, so test should work?

Reply:
Good that you saw this! I referenced the wrong issue, fixed now (it's actually #3588).